subsys/net/lib/wifi_credentials/Kconfig

```diff
@@ -57,4 +63,16 @@ config WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET
 	  This library will use WIFI_CREDENTIALS_MAX_ENTRIES entries starting with the
 	  WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET uid.
 
+
 endif # WIFI_CREDENTIALS_BACKEND_PSA
+
+config WIFI_CREDENTIALS_STATIC_SSID
+	string "SSID of statically configured WiFi network"
+
+config WIFI_CREDENTIALS_STATIC_PASSWORD
+	string "Password of statically configured WiFi network"
+	default ""
+
+config WIFI_CREDENTIALS_STATIC_TYPE
```

Comment:
Wouldn't it be better to use `choice` type Kconfig entry, so that users don't have a chance to misconfigure by providing an unsupported value (string) here?

Reply:
done